File: e006.py
"""
Problem 6: Find the difference between the sum of the squares of the
first one hundred natural numbers and the square of the sum.
"""

natural_numbers = range(1, 101)
a = sum([i ** 2 for i in natural_numbers])
b = sum([i for i in natural_numbers])
print(b ** 2 - a)

File: e006.js
// import {sum} from "./myfunc.js";
const sum = (arr) => arr.reduce((s, i) => s + i, 0);

const naturalNumbers = [];
for (let i = 1; i <= 100; i++) {
    naturalNumbers.push(i);
}

const a = sum(naturalNumbers.map(i => i ** 2));
const b = sum(naturalNumbers);

let answer = b ** 2 - a;
console.log(answer);
